fili-generic-example/src/main/java/com/yahoo/wiki/webservice/data/config/auto/DruidNavigator.java

```diff
@@ -234,32 +233,34 @@ private void loadTimeGrains(TableConfig tableConfig, JsonNode segmentJson) {
         }
 
         if (!timeGrain.isPresent()) {
-            LOG.warn("Couldn't detect timegrain for {}, defaulting to DAY TimeGrain.", timeInterval.asText());
+            LOG.warn("Couldn't detect timegrain for {}, defaulting to DAY TimeGrain.", timeInterval);
         }
         tableConfig.setTimeGrain(timeGrain.orElse(DefaultTimeGrain.DAY));
     }
 
     /**
-     * Send a blocking request to druid. All queries are finished before continuing.
+     * Send a non-blocking request to Druid.
```

Comment:
Having traced back in git history, I think https://github.com/yahoo/fili/pull/309 changes this method(`queryDruid`) from blocking to non-blocking but forgot to update JavaDoc